components/dashboard/src/start/StartPage.tsx

```diff
@@ -105,6 +107,17 @@ export function StartPage(props: StartPageProps) {
                 )}
                 {error && <StartError error={error} />}
                 {props.children}
+                {props.showLatestIdeWarning && (
+                    <InfoBox className="mt-4">
+                        You are using the latest editor version.{" "}
+                        <a
```

Comment:
**issue**: Since we now have the link to user preferences included here we could remove the notice below action buttons that also links to user preferences. This will become redundant with upcoming changes like https://github.com/gitpod-io/gitpod/issues/6910. See also relevant [design exploration](https://www.figma.com/file/3K7TIuko6ADxO0FY941huB/Display-warning-for-latest-IDE-versions-on-the-loading-screen-%238028?node-id=1111%3A508).

Referring to the following element:
> These IDE options are based on your user preferences.

Alternatively, we could replace this line with an info alert component like the following which could be shown only on the running workspace start phase for now. What do you think?

| Info alert component |
|-|
| <img width="1440" alt="Running" src="https://user-images.githubusercontent.com/120486/158706097-11d0d26e-c90c-4572-8a0f-3513b3955ce6.png"> |